crates/cargo-wdk/src/actions/build.rs

```diff
@@ -30,7 +33,8 @@ pub enum BuildActionError {
 /// Action that orchestrates building of driver project using cargo command.
 pub struct BuildAction<'a> {
     package_name: &'a str,
-    profile: &'a Profile,
+    profile: Option<Profile>,
```

Comment:
Since we have a lifetime declared on this struct, we could pass Option<&'a Profile> here.